package.json

```diff
@@ -48,6 +48,7 @@
     "benchmark": "^2.1.4",
     "bili": "^4.8.1",
     "chalk": "^2.4.2",
+    "core-js": "^2",
```

Comment:
As this is a dev only dep, can we also upgrade to core-js@3?

Reply:
Will do it